src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
```

Comment:
Define CSS variables for repeated values like colors and font families to make future changes easier.

```
:root {
  --main-color: #00acdc;
  --font-family: "Roboto", sans-serif;
  --link-color: #000;
}
```